### PR TITLE
Exclude fragment anchors

### LINKED ISSUE #4: is it possible to exclude fragment anchors?
This plugin is great.
Thanks for making it.

Is it possible to exclude fragment anchors?
As it is now, all my fragment links are getting the external link treatment.

image

thanks!

### PR DESCRIPTION
**Pull request by Murderlon**
fixes #4

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -3,23 +3,16 @@
...
   "description": "Automatically adds the target and rel attributes to external links",
   "license": "MIT",
-  "keywords": [
-    "markdown",
-    "remark",
-    "plugin",
-    "link",
-    "url"
-  ],
+  "keywords": ["markdown", "remark", "plugin", "link", "url"],
```
Comment (wooorm) [inline package.json]: Could you drop these unnecessary style changes, except for adding your name?

**Review by wooorm — DISMISSED** [review]
Looks good, only one minor nit!

File: package.json (hunk 1 above)
Comment (Murderlon) [reply package.json]: Ugh Prettier!

**Comment by zslabs** [comment]
Thanks for this! Would you be able to release a new version? We actually just noticed this issue on our end and would love to upgrade to resolve. Thanks again.

**Comment by Murderlon** [comment]
@zslabs just published the release :)

**Comment by zslabs** [comment]
You 'da best! Appreciate it!

**Pull request merged**